openpype/hosts/maya/plugins/publish/validate_look_color_space.py

```diff
@@ -0,0 +1,25 @@
+from maya import cmds
+
+import pyblish.api
+from openpype.pipeline.publish import ValidateContentsOrder
+
+
+class ValidateMayaColorSpace(pyblish.api.InstancePlugin):
+    """
+        Check if the OCIO Color Management and maketx options
+        enabled at the same time
+    """
+
+    order = ValidateContentsOrder
+    families = ['look']
+    hosts = ['maya']
+    label = 'Maya Color Space'
+
+    def process(self, instance):
+        ocio_maya = cmds.colorManagementPrefs(q=True,
+                                              cmConfigFileEnabled=True,
+                                              cmEnabled=True)
+        maketx = instance.data["maketx"]
+
+        if ocio_maya and maketx:
+            raise Exception("Maya is color managed and maketx option is on. OpenPype doesn't support this combination yet.") # noqa
```

Comment:
This should likely be a more specific exception than a bare `Exception` like e.g. `RuntimeError`

But, even better - to prepare for future publisher I'd already turn this into a `KnownPublishError` instead.